File: app/src/main/java/com/mx/easytouch/vo/InstallPackage.java
package com.mx.easytouch.vo;

/**
 * Created by maoxin on 2017/11/13.
 */

public  class InstallPackage{

    private String appName;
    private String packageName;
    private int id;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public InstallPackage(){
    }

    public InstallPackage(String appName, String packageName){
        this.appName = appName;
        this.packageName = packageName;
    }

    public String getAppName() {
        return appName;
    }

    public void setAppName(String appName) {
        this.appName = appName;
    }

    public String getPackageName() {
        return packageName;
    }

    public void setPackageName(String packageName) {
        this.packageName = packageName;
    }
}

File: app/src/main/java/com/mx/easytouch/vo/InstallPackage.kt
package com.mx.easytouch.vo

import android.os.Parcel
import android.os.Parcelable

/**
 * Created by maoxin on 2017/11/13.
 */

class InstallPackage constructor(var appName: String?, var packageName: String?):Parcelable {

    var id: Int = 0

    constructor(parcel: Parcel) : this(
            parcel.readString(),
            parcel.readString()) {
        id = parcel.readInt()
    }

    constructor() : this(
           null,
            null)

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(appName)
        parcel.writeString(packageName)
        parcel.writeInt(id)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<InstallPackage> {
        override fun createFromParcel(parcel: Parcel): InstallPackage {
            return InstallPackage(parcel)
        }

        override fun newArray(size: Int): Array<InstallPackage?> {
            return arrayOfNulls(size)
        }
    }

}
